### DISCUSSION TITLE
Escaping " or '

### DISCUSSION THREAD

**Discussion by cezamee** [BishopFox/sliver#1602] [Q&A] [answered]
Hey guys, is their any way to escape " or ' while using execute command, for example echoing something to a file ?

execute -o echo "@reboot sleep 300 && /to/smth" | crontab -

**Comment by moloch--** [comment, marked as answer]
You should be able to use `--` e.g., `execute -- <anything>` note however depending on the situation `execute` may not invoke a shell, and thus shell operators may not function as you expect them to i.e., you may need to do something akin to `execute -- /bin/bash -c "foobar"`

**Reply by cezamee** [reply]
Thank you for the explanations, I appreciate it.
